pkg/network/netlink/consumer_test.go

```diff
@@ -81,13 +80,11 @@ func TestConsumerKeepsRunningAfterCircuitBreakerTrip(t *testing.T) {
 	if pre315Kernel {
 		require.Eventually(t, func() bool {
 			return !isRecvLoopRunning() && c.breaker.IsOpen()
-		}, 3*time.Second, 100*time.Millisecond)
+		}, cfg.ConntrackRateLimitInterval, 100*time.Millisecond)
```

Comment:
If we are keeping this == to the tick interval, how does decreasing the tick interval make this less flaky? I would think we would keep this > the tick interval?

Reply:
The problem was that the previous code wasn't actually generating netlink messages. Changing to using cross-namespace connections is the real fix here.

Reply:
No need